src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -179,12 +179,24 @@ private void applyStringFilter(String filterString) {
             } else {
                 this.applyFilterExpression(Qualifier.EMPTY);
             }
-        } catch (ParseException ex) {
+        } catch (FilterException ex) {
             this.applyFilterExpression(Qualifier.EMPTY);
             // Overrides message in status bar
-            UI.status.displayMessage("Panel " + (panelIndex + 1)
-                + ": Parse error in filter: " + ex.getMessage());
+            UI.status.displayMessage(resolvePanelName(panelMenuBar.getPanelName())
+                + ": " + ex.getMessage());
+        }
+    }
+
+    /**
+     * Shows panel name if renamed by user. Panel index is shown otherwise.
+     * @param panelName
+     * @return final panel name shown to users
+     */
+    private String resolvePanelName(String panelName) {
+        if (panelName.equals(PanelMenuBar.DEFAULT_PANEL_NAME)) {
+            return PanelMenuBar.DEFAULT_PANEL_NAME + " " + (panelIndex + 1);
```

Comment:
And this one `panelIndex`. Try to give all the "inputs" the method need as parameters. That makes the method signature more self explanatory.